doc/source/tutorials/tutorial-level-5-migration.rst

```diff
@@ -185,18 +197,24 @@ The package will most likely have failed ``pre-commit`` hooks. We will manually
 
 Here, instead of fixing all errors at once, we will address each type of error one at a time.  For example, a branch called ``pre-commit-spelling`` may contain spelling fixes, while another branch, ``pre-commit-flake8-line`` fixes of line length errors raised by flake8.
 
-#. Sync with the ``upstream/migration`` branch:
+#. To get the ``migration`` branch from ``upstream/migration`` and track it, run the following commands:
+
+    .. code-block:: bash
+
+        git fetch upstream migration
+        git checkout --track upstream/migration
```

Comment:
Continuing our discussion here - I note that local `migration` branch doesn't exist. I

 think `git checkout upstream migration` will do and it will automatically track it? I didn't have to write `--track` in my experience.

Reply:
Okay. I see prev discussion from the "outdated" comments.

Reply:
OK, thnaks @bobleesj let's keep it then?